qubit.py

```diff
@@ -0,0 +1,66 @@
+
+import numpy as np
+import math as m
+import random 
+
+class Qubit(object):
+    
+    def __init__(self, alpha, beta, theta, mag):
```

Comment:
Does this work if you only want to provide alpha and beta, or when you only want to use the polar representation? 
I think you need to add both the default values and then some checks that either alpha and beta have been inputed, or that theta and mag have been inputed.

Reply:
We weren't sure how to check. Is it something like if alpha == None etc ?